Vagrantfile

```diff
@@ -48,6 +48,7 @@ vvv_config['sites'].each do |site, args|
   defaults['branch'] = 'master'
   defaults['skip_provisioning'] = false
   defaults['allow_customfile'] = false
+  defaults['upstream'] = 'php'
```

Comment:
I decided on `upstream` instead of `php_version` as i realised it could be used for so much more than PHP, just as a test i have already run it with Node & Python upstreams.